code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1891,7 +1891,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/monkeysdelight/New()
 	..()
-	reagents.add_reagent(NUTRIMENT, 10)
+	reagents.add_reagent(NUTRIMENT, 15)
```

Comment:
Ah, fair enough.

Raisins are a common reference point that we're all familiar with to put the numbers in scale.